### PR TITLE
Form for quick object comments

### PR DESCRIPTION
**Pull request by aweiksnar**
- Quick last minute hacking for #660, I tried to keep most of the terribleness to `quick-subject-comment-form.cjsx` so it can be refactored later
- Adds a form for adding subject comments to a default board - titled 'Notes', but editable
- Mods need to click a "Activate Talk Subject Comments Board" in the mod controls on a talk boards page in a project to create the default board for subject comments, since they are the only ones who can create boards
- After that there should be 2 tabs on the bottom of the object page, one to leave a comment in the subject's discussion in the default board, and one to start a new discussion
- I'll keep pushing to this branch until we deploy

### CODE REVIEW

File: app/talk/quick-subject-comment-form.cjsx [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+                    discussion = {
+                      title: "Subject #{@props.subject.id}"
+                      user_id: @props.user?.id
+                      subject_default: true,
+                      board_id: board.id
+                      comments: comments
+                      }
+                    talkClient.type('discussions').create(discussion).save()
+                      .then (discussion) =>
+                        @transitionTo('project-talk-discussion', {owner: @props.user?.login, name: project.slug, board: discussion.board_id, discussion: discussion.id})
```
Comment (chrissnyder) [inline app/talk/quick-subject-comment-form.cjsx]: Should this be the project owner rather than the user?

Comment (aweiksnar) [inline app/talk/quick-subject-comment-form.cjsx]: Yep thanks just updated

File: app/talk/quick-subject-comment-form.cjsx [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+                    user_id = @props.user?.id
+                    body = commentText
+                    discussion_id = discussion.id
+
+                    comment = merge {}, {user_id, discussion_id, body}
+
+                    talkClient.type('comments').create(comment).save()
+                      .then (comment) =>
+                        pageOfComment = getPageOfComment(comment, discussion, PAGE_SIZE)
+                        @transitionTo('project-talk-discussion', {owner: @props.user?.login, name: project.slug, board: discussion.board_id, discussion: discussion.id}, {page: pageOfComment, comment: comment.id})
```
Comment (camallen) [inline app/talk/quick-subject-comment-form.cjsx]: I still get redirected incorrectly - is the above transition correct?

Comment (aweiksnar) [inline app/talk/quick-subject-comment-form.cjsx]: Sorry missed that one, Just updated should be the correct redirect now

**Comment by chrissnyder** [comment]
LGTM

**Comment by camallen** [comment]
LGTM

**Comment by edpaget** [comment]
:ship:

**Pull request merged**